player.html

```diff
@@ -6,8 +6,16 @@
     body{
       background-color:black;
     }
+    /* https://chromium.googlesource.com/chromium/blink/+/master/Source/core/css/mediaControlsNew.css#89 */
     #video{
-      width:100%;
+      position: absolute;
+      top: 0px;
+      right: 0px;
+      bottom: 0px;
+      left: 0px;
+      margin: auto;
+      max-height: 100%;

```

Comment:
Nice ! Is that centering the video inside the tab? Does the video change size as it switches bitrates?